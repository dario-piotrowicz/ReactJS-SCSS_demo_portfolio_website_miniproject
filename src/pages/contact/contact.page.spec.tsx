import React from 'react';
import { shallow } from 'enzyme';
import ContactPage from './contact.page.component';

describe('contact page', () => {
  it('should match its snapshot', () => {
    expect(shallow(<ContactPage />)).toMatchSnapshot();
  });
});
