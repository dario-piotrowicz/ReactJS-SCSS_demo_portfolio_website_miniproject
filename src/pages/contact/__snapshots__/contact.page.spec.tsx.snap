// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contact page should match its snapshot 1`] = `
<Fragment>
  <ContactForm />
  <ContactInfoSection />
  <ContactTagline />
</Fragment>
`;
